enhancements/ota/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
nit: I expect "ota" to be fairly opaque to external folks, and would prefer `enhancements/update/...` (which is what I'm using in #124).